worker/worker.go

```diff
@@ -29,6 +31,24 @@ type Worker struct {
 	QueueId        string
 }
 
+type TaskState uint8
+
+const (
+	TaskPending TaskState = iota
+	TaskRunning
+	TaskFinished
+	TaskFailed
+)
+
+type Task struct {
+	Commands       []string
+	ReportInterval int64
+	State          TaskState
+	Progress       int
+	Image string
```

Comment:
what is this Image?

Reply:
the container image the task must be executed. I'll add a doc for this too

Reply:
is there a name to this abstract on docker? maybe, "docker container name"? Also, the point of doc is to ease the life of the dev is supposed to use the code, e.g can I add a tag or digest into this name?

Reply:
... "docker image name"

Reply:
I think DockerImage sounds ok, can it be?